### ISSUE TITLE
can not reproduce the results following the hyparameters in the paper

### ISSUE THREAD

**Issue by junleiz** [dropreg/R-Drop#23]
Hello, I tired to reproduce the GLUE results following the hyperparameters in the paper:
![image](https://user-images.githubusercontent.com/26025961/152580992-be6ecc32-c067-46cc-83ba-f6201f98898f.png)
I added the alpha in the following lines for bert:
![image](https://user-images.githubusercontent.com/26025961/152581065-89ce541b-b2a4-450e-8e42-0eff227d4bd8.png)
However, some results are far away from the reported in the paper:

my results:
![image](https://user-images.githubusercontent.com/26025961/152581175-03e72f81-b433-4efd-bc37-36e1ae940498.png)
paper:
![image](https://user-images.githubusercontent.com/26025961/152581246-bfc33f2c-3d9a-46c5-b036-6b4e86c3f90e.png)

**Comment by dropreg** [comment]
please refer to #6 

Many GLUE experiments require some specific “random seed“ and hyper-parameters. However, from the experimental results, R-Drop can be guaranteed to be better than the original fine-tuining method.
